providers/Flipt/LICENSE

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```

Comment:
mind updating this to MIT? its what we use for most of our client sdks

Reply:
Apache 2 is the recommended license for cncf projects.

Reply:
Thanks @beeme1mr sounds good